src/rows.c

```diff
@@ -113,20 +122,21 @@ size_t max_token_len_with_converters(char32_t **tokens, int num_tokens,
         }
 
         if (conv_funcs && conv_funcs[j]) {
-            PyObject *obj = call_converter_function(conv_funcs[i], tokens[j]);
+            PyObject *obj = call_converter_function(conv_funcs[i], tokens[j],
+                                                    byte_converters);
             if (obj == NULL) {
-                fprintf(stderr, "CALL FAILED!\n");
+                return NULL;
```

Comment:
```suggestion
                return -1;
```
and all others.  A static should find this (not that it matters), but I guess they are almost impossible to cover.